es6-promises/index.js

```diff
@@ -0,0 +1,11 @@
+const takeAChance = require('./take-a-chance');
+
+const randomLuck = takeAChance('Adrian');
+
+randomLuck.then(value => {
+  console.log(value);
+});
+
+randomLuck.catch(error => {
+  console.error(error.message);
+});
```

Comment:
This will work, but traditionally we chain these together. For example:
```JAVASCRIPT
randomLuck.then(value => {
  console.log(value);
}).catch(error => {
  console.error(error.message);
});
```